oid4vci/pyproject.toml

```diff
@@ -0,0 +1,88 @@
+[tool.poetry]
+name = "oid4vci"
+version = "0.1.0"
+description = "oid4vci plugin for acapy."
+authors = ["Adam Burdett <burdettadam@gmail.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.9"
+aiohttp = "^3.8.5"
+aries-cloudagent = { version = "0.11.0" }
+aries-askar = "~0.3.0"
+aiohttp-cors = "^0.7.0"
+marshmallow = "^3.20.1"
+
+[tool.poetry.dev-dependencies]
+ruff="^0.0.285"
+black="23.7.0"
+
+# testing
+asynctest = "0.13.0"
+pytest = "~7.4.0"
+pytest-asyncio = "0.14.0"
+pytest-ruff="^0.1.1"
+mock= "~4.0"
+
+[tool.poetry.group.integration.dependencies]
+indy-credx= { version = "~1.0.0" }
+indy-vdr= { version = "~0.3.3" }
+
+
+[tool.poetry.group.dev.dependencies]
+pre-commit = "^3.5.0"
+pytest-cov = "^4.1.0"
+
+[tool.ruff]
+select = ["E", "F", "C", "D"]
+
+ignore = [
+    # Google Python Doc Style
+    "D203", "D204", "D213", "D215", "D400", "D401", "D404", "D406", "D407",
+    "D408", "D409", "D413",
+    "D202", # Allow blank line after docstring
+    "D104", # Don't require docstring in public package
+    # Things that we should fix, but are too much work right now
+    "D417", "C901",
+]
+
+line-length = 90
+
+[tool.ruff.per-file-ignores]
+"**/{tests}/*" = ["F841", "D", "E501"]
+
+[tool.pytest.ini_options]
+testpaths = "basicmessage_storage"
```

Comment:
Copy and paste error here. I think changing this and the integration tests folder name would get the unit tests succeeding.